feat: polygon parts documentation (#76)
MapColonies/developer-portal: docs/MapColonies/Raster/services/PolygonParts/profile_v1_0.md

---
id: polygon-parts-profile-v1
slug: polygon-parts-profile-v1
description: v1 Profile for polygon parts attributes
title: polygon-parts-profile-v1 🧩
tags:
  - raster
  - polygon parts
  - v1
  - wfs
---

# Polygon Parts Attributes Information

| **Polygon Part Queryable <br/> Attribute Name** | **Type** | **Description** | 
| ----------- | ----------- | ----------- | 
| geometry | GeoJson (geometry (Polygon,4326)) | Feature's geometry |

> | partId | integer | Internal unique polygon part identifier (1-2147483647) |

Reviewer: ```suggestion
| partId | integer | Internal unique part identifier (1-2147483647) |
```

> | recordId | UUID | The original record (layer) identifier in the catalog DB |

Reviewer: ```suggestion
| recordId | UUID | Mapcolonies reaster catalog identifier for the layer |
```

| productId | text | Product id of the layer |
| productType | text | Product type of the layer |
| id | text | External Id of the polygon part |
| name | text | Product name of the polygon part |
| updatedInVersion | text | Product version of the layer |
| ingestionDateUTC | date | The time this part was ingested |
| imagingTimeBeginUTC | date | Start imaging date (UTC time) of polygon parts |
| imagingTimeEndUTC | date | End imaging date (UTC time) of polygon parts |
| resolutionDegree | number | Polygon part ingestion resolution degree (1.67638e-7 to 0.703125) |
| resolutionMeter | number | Polygon part ingestion resolution meter (1.67638e-7 to 0.703125) |

> | sourceResolutionMeter | number | Polygon part source resolution meter (0.0185 to 78273) |

Reviewer: Why isn't there `sourceResolutionDegree`?

Author: ask Shaziri, it's according to the fields that were designed:
https://mapcolonies.atlassian.net/wiki/spaces/MAPConflicResolution/pages/1869283380/Polygon-Parts+query+fields+and+DB+structure

> | sourceResolutionMeter | number | Polygon part source resolution meter (0.0185 to 78273) |

Reviewer: ```suggestion
| sourceResolutionMeter | number | Part source resolution meter (0.0185 to 78273) |
```

> | horizontalAccuracyCE90 | number | Polygon part accuracy (0.01 to 4000) |

Reviewer: ```suggestion
| horizontalAccuracyCE90 | number | Part horizontal accuracy (0.01 to 4000) |
```

> | sensors | text (comma delimited) | Polygon part sensors array |

Reviewer: ```suggestion
| sensors | text (comma delimited) | The sensors used to create the part |
```

> | countries | text (comma delimited) | Polygon part countries array |

Reviewer: ```suggestion
| countries | text (comma delimited) | All of the countries that the part has data in |
```

| cities | text  (comma delimited) | All of the cities that the part has data in |
| description | text | Part description |

> 

Reviewer 2: add valid example response

Reviewer: I don't think this is relevant just like there is no response example for the `raster` catalog.

Author: the profile explains each attribute the polygon part feature should include.

exactly for this purpose the protocol of `DescribeFeatureType` shows in details those attributes, and the guide already shows such standard request & response

> | cities | text  (comma delimited) | Polygon part cities array |

Reviewer: ```suggestion
| cities | text  (comma delimited) | All of the cities that the part has data in |
```

> | description | text | As was inserted by catalog manager |

Reviewer: ```suggestion
| description | text | Part description |
```
